connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -82,6 +87,11 @@ public class ConnectorConfig extends AbstractConfig {
                     " or use \"FileStreamSink\" or \"FileStreamSinkConnector\" to make the configuration a bit shorter";
     private static final String CONNECTOR_CLASS_DISPLAY = "Connector class";
 
+    public static final String CONNECTOR_VERSION = "connector." + WorkerConfig.PLUGIN_VERSION_SUFFIX;
```

Comment:
@snehashisp Thanks for this great patch. I have one small comment: It seems we are using `connector.plugin.version` instead of `connector.version`, correct? If so, could you please update the KIP accordingly?